### DISCUSSION TITLE
Install using bun

### DISCUSSION THREAD

**Discussion by MateoWartelle** [ant-design/ant-design#48261] [General]
### What problem does this feature solve?

Share an additional snippet for users who use Bun

### What does the proposed API look like?

https://ant.design/docs/react/introduce#using-npm-or-yarn-or-pnpm - Add another tab for Bun

**Comment by afc163** [comment]
PR is welcome~

**Reply by kiner-tang** [reply]
- https://github.com/ant-design/ant-design/pull/48289

**Reply by MateoWartelle** [reply]
Thank you, I appreciate.
